src/ringmesh/visualize/mesh_entity_gfx.cpp

```diff
@@ -113,16 +111,17 @@ namespace RINGMesh
 
         index_t nb_coordinates() override
         {
-            GEO::AttributeStore* store =
+            AttributeStore* store =
                 get_attribute_manager().find_attribute_store(
                     this->manager_->name() );
 
             if( store == nullptr )
                 return 0;
-            return store->dimension();
+            return 1;
+            //            return store->get_store().dimension();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ACommentedCode)